buildpacks/jvm/CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.1.8] 2021/08/25
```

Comment:
The bot will add the version/date when the release occurs (which is performed via a GitHub action rather than a PR). Could you omit this bit for now? :-)

Reply:
Sorry, just saw this comment! Updated the PR, thank you!